index.html

```diff
@@ -2237,40 +2237,40 @@ <h4>
 
     <section id="mixed_text_composition_in_vertical_writing_mode">
       <h4>
-        <span its-locale-filter-list="en" lang="en">Mixed Text Composition in Vertical Writing Mode</span>
+        <span its-locale-filter-list="en" lang="en">Mixed text Composition in Vertical Writing Mode</span>
```

Comment:
Why is 'Text' changed to lowercase here?

Reply:
The “text” [sic] here has not been modified. In this PR, only the “Western” was changed into “Mixed”. See ec4f3837.